tests/acceptance/custom-user-test.js

```diff
@@ -0,0 +1,38 @@
+import { test, module } from 'qunit';
+import startApp from '../helpers/start-app';
+import destroyApp from '../helpers/destroy-app';
+import config from 'dummy/config/environment';
+import Bugsnag from 'bugsnag';
+
+module('Acceptance | custom user', {
+  beforeEach() {
+    config.bugsnag.releaseStage = 'production';
+    this.cachedEnvValue = config.environment; // NOTE: cached so we can restore it
+    config.environment = 'addon-test';
+    this.application = startApp();
+  },
+
+  afterEach() {
+    delete config.bugsnag.releaseStage;
+    config.environment = this.cachedEnvValue; // NOTE: restore original value
+    destroyApp(this.application);
+  }
+});
+
+test('with custom user set', function(assert) {
+  visit('/');
+  click('button');
+
+  andThen(function() {
+    const expected = {
+      id: 123,
+      name: 'Dummy User',
+      email: 'dummy@example.com',
+    };
+    assert.deepEqual(
```

Comment:
We may want to get more fancy here to test different types of configuration since at the moment the test util is hardcoded.